gradle/root/coverage.gradle

```diff
@@ -1,15 +1,27 @@
-apply plugin: 'jacoco'
+//TODO: Standard JaCoCo coverage doesn't work in Android module
+task mockitoCoverage(type: JacocoReport) {
 
-// TODO later include subprojects, possible ideas https://gist.github.com/aalmiray/e6f54aa4b3803be0bcac
+    allprojects { currentProject ->
+        plugins.withId("java") {
```

Comment:
Should we also take into account the Kotlin projects, as they are also generating coverage?

Reply:
The groovy/kotlin plugins applies the `java` plugin under the hood.

Reply:
Ah okay :+1: